.github/workflows/TagReminder.yml

```diff
@@ -0,0 +1,20 @@
+name: "Reminder"
+on:
+  # works for mannual, in case of github agent updated. 
+  # we can run mannually to see if it can be fixed by update version of
+  # - kubectl
+  # - k8s cluster provider's 
+  schedule: [{cron: "0 0 1 */3 *"}]
+
+jobs:
+  versionBumpUpReminder:
+    name: versionBumpUpReminder
+    runs-on: ubuntu-latest
+    steps:
+    - name: Create new issue
+      uses: imjohnbo/issue-bot@v3
+      with:
+        title: AutoBumpUpVersionReminder
+        body: |-
+          :wave: Hi maintainers, please remind to bump version for kind, microshift or any dependency for github action! 
```

Comment:
what about having script that detecting latest kind, microshift or other dependency and creating an issue?

Reply:
in fact kind is provided by GHA, microshift seems don't have a release follow up with v.x.x.x. format, hence for now, a full automatic maybe too early?
btw, if we do so, what's different with dep bot?

hence my plan is to use/replay on dep bot to create pr in automatic for each dependency. and use this action to bump version of local-dev-cluster per 3 months.